bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerOpenOp.java

```diff
@@ -207,10 +207,19 @@ public void safeOperationComplete(int rc, Void result) {
                             if (ex != null) {
                                 LOG.error("Ledger {} close failed", ledgerId, ex);
                             }
-                            openComplete(BKException.Code.UnauthorizedAccessException, null);
+                            openComplete(rc, null);
+                        });
+                    } else if (rc == BKException.Code.TimeoutException) {
+                        closeLedgerHandleAsync().whenComplete((r, ex) -> {
+                            if (ex != null) {
+                                LOG.error("Ledger {} read timeout", ledgerId, ex);
+                            }
+                            openComplete(rc, null);
                         });
```

Comment:
```java
LOG.error("Ledger {} read timeout", ledgerId, ex);
```
should be replaced with the following:
```java
LOG.error("Ledger {} close failed", ledgerId, ex);
```

the `ex` is not the read timeout exception.

Reply:
You are right, already fixed